subprojects/docs/src/docs/userguide/introductionDependencyManagement.adoc

```diff
@@ -50,7 +62,7 @@ Gradle takes your dependency declarations and repository definitions and attempt
 
 * Given a required dependency, Gradle first attempts to resolve the _module_ for that dependency. Each repository is inspected in order, searching first for a _module descriptor_ file (POM or Ivy file) that indicates the presence of that module. If no module descriptor is found, Gradle will search for the presence of the primary _module artifact_ file indicating that the module exists in the repository.
 
-** If the dependency is declared as a dynamic version (like `1.+`), Gradle will resolve this to the newest available static version (like `1.2`) in the repository. For Maven repositories, this is done using the `maven-metadata.xml` file, while for Ivy repositories this is done by directory listing.
+** If the dependency is declared as a dynamic version (like `1.+`), Gradle will resolve this to the highest available static version (like `1.2`) in the repository. For Maven repositories, this is done using the `maven-metadata.xml` file, while for Ivy repositories this is done by directory listing.
 
 ** If the module descriptor is a POM file that has a parent POM declared, Gradle will recursively attempt to resolve each of the parent modules for the POM.
```

Comment:
module descriptor -> module metadata file

I think it's the better term for describing what this is, because people usually can relate to the term 'metadata'. If we change it, change it everywhere in the document.

Reply:
Changed to "module metadata". I'll need to make a sweep through all of the content once we settled on the terminology. I'll do that in a separate PR.